client/index.js

```diff
@@ -30,7 +30,10 @@ const {
   location
 } = window
 
-// With this, static assets will work across zones
+// With dynamic assetPrefix it's no longer possible to set assetPrefix at the build time
+// So, this is how we do it in the client side at runtime
+__webpack_public_path__ = `${assetPrefix}/_next/webpack/` //eslint-disable-line
```

Comment:
@timneutkens have a look at this.
This is a public API: https://webpack.js.org/guides/public-path/#on-the-fly

Reply:
🕵️